plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java

```diff
@@ -235,7 +235,6 @@ private Class getComponentType(TypePair type) {
                 if (superclass instanceof ParameterizedType) {
                     final ParameterizedType psc = (ParameterizedType) superclass;
                     t = psc.getActualTypeArguments()[0];
-                    continue;
```

Comment:
This change might have unexpected effect.

Reply:
Reading this again, this has no side effects. I was lacking coffee 🗡️👁️
Feel free to add the removal back.

Reply:
Done, dropping a single commit isn't that complicated :-)